### ISSUE TITLE
Dependencies unmet when installing grunt-cli

### ISSUE THREAD

**Issue by Wilkins-** [gruntjs/grunt-cli#41]
OSX 10.7
Node.js v0.10.24

When installing grunt-cli on my Mac I am getting the following errors (dependency unmet):

```
sudo npm install -g grunt-cli
npm http GET https://registry.npmjs.org/grunt-cli
npm http 304 https://registry.npmjs.org/grunt-cli
npm http GET https://registry.npmjs.org/nopt
npm http GET https://registry.npmjs.org/findup-sync
npm http GET https://registry.npmjs.org/resolve
npm http 304 https://registry.npmjs.org/findup-sync
npm http 304 https://registry.npmjs.org/resolve
npm http 304 https://registry.npmjs.org/nopt
npm http GET https://registry.npmjs.org/abbrev
npm http GET https://registry.npmjs.org/glob
npm http GET https://registry.npmjs.org/lodash
npm http 304 https://registry.npmjs.org/abbrev
npm http 304 https://registry.npmjs.org/glob
npm http 304 https://registry.npmjs.org/lodash
npm http GET https://registry.npmjs.org/minimatch
npm http GET https://registry.npmjs.org/graceful-fs
npm http GET https://registry.npmjs.org/inherits
npm http 304 https://registry.npmjs.org/minimatch
npm http 304 https://registry.npmjs.org/graceful-fs
npm http 304 https://registry.npmjs.org/inherits
npm http GET https://registry.npmjs.org/lru-cache
npm http GET https://registry.npmjs.org/sigmund
npm http 304 https://registry.npmjs.org/lru-cache
npm http 304 https://registry.npmjs.org/sigmund
/usr/local/share/npm/bin/grunt -> /usr/local/share/npm/lib/node_modules/grunt-cli/bin/grunt
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/bower requires colors@'latest' but will load
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/colors,
npm WARN unmet dependency which is version 0.6.0-1
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/bower requires rimraf@'latest' but will load
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/rimraf,
npm WARN unmet dependency which is version 2.0.2
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/bower requires mkdirp@'latest' but will load
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/mkdirp,
npm WARN unmet dependency which is version 0.3.4
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/bower requires fstream@'latest' but will load
npm WARN unmet dependency /usr/local/share/npm/lib/node_modules/yeoman/node_modules/fstream,
npm WARN unmet dependency which is version 0.1.19
grunt-cli@0.1.11 /usr/local/share/npm/lib/node_modules/grunt-cli
├── resolve@0.3.1
├── nopt@1.0.10 (abbrev@1.0.4)
└── findup-sync@0.1.2 (lodash@1.0.1, glob@3.1.21)

```

**Comment by victorsj1** [comment]
I can't find any documentation on resolving these dependencies. Getting frustrated.

**Comment by shama** [comment]
@victorsj1 Just ignore those warnings or uninstall the global module causing them. For instance, in the above example, it's `yeoman` (and not `grunt-cli`) that has peer dependencies which cause warnings. So you can do `npm uninstall yeoman -g` to remove it.

TL;DR, peer dependencies are awful. Either ignore the warnings, uninstall the modules that use peer dependencies, or file an issue on the problem projects about the peer deps (if it's yeoman, that project looks deprecated now).
